otel-loadbalancer/discovery/testdata/test.yaml

```diff
@@ -0,0 +1,12 @@
+mode: LeastConnection
+label_selector:
+  app.kubernetes.io/instance: default.test
+  app.kubernetes.io/managed-by: opentelemetry-operator
+config:
+  scrape_configs:
+  - job_name: prometheus
+
+    static_configs:
+    - targets: ["prom.domain:9001", "prom.domain:9002", "prom.domain:9003"]
+      labels:
+        my: label
```

Comment:
The test case for file_sd_config was added just to showcase the property of discovery manager in discovering _***_sd_configs**_ since static configs can be directly retrieved from the original yaml file. Having just the static configs may not fully help in testing the "discovery" capabilities. Just my thinking and I may be completely wrong here.